src/app/(dashboard)/agent-network/usage/page.tsx

```
@@ -59,8 +72,8 @@ export default function UsageAndLogsPage() {
   // Also reset to the default when ?tab= is removed or invalid, so navigating
   // back to the bare URL doesn't leave the previous tab selected.
   useEffect(() => {
-    setTab(VALID_TABS.has(queryTab) ? queryTab : TAB_USAGE);
-  }, [queryTab]);
+    setTab(allowedTabs.has(queryTab) ? queryTab : defaultTab);
```

ESLint

[error] Error: Calling setState synchronously within an effect can trigger cascading renders

Effects are intended to synchronize state between React and external systems such as manually updating the DOM, state management libraries, or other platform APIs. In general, the body of an effect should do one or both of the following:
* Update external systems with the latest state from React.
* Subscribe for updates from some external system, calling setState in a callback function when external state changes.

Calling setState synchronously within an effect body causes cascading renders that can hurt performance, and is not recommended. (https://react.dev/learn/you-might-not-need-an-effect).

/home/jailuser/git/src/app/(dashboard)/agent-network/usage/page.tsx:75:5
  73 |   // back to the bare URL doesn't leave the previous tab selected.
  74 |   useEffect(() => {
> 75 |     setTab(allowedTabs.has(queryTab) ? queryTab : defaultTab);
     |     ^^^^^^ Avoid calling setState() directly within an effect
  76 |   }, [queryTab, allowedTabs, defaultTab]);
  77 |
  78 |   // Reflect the active tab in the URL so it's shareable, like Settings.

(react-hooks/set-state-in-effect)